webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -260,13 +260,13 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
                             <tr title="{{ fields_desc['raw_crash.AdapterVendorID'] }}">
                                 <th scope="row">Adapter Vendor ID</th>
                                 <td>
-                                    <pre>{{ raw.AdapterVendorID }}</pre>
+                                    <pre>{{ raw.AdapterVendorID }}{% if raw.AdapterVendorName %} ({{ raw.AdapterVendorName }}){% endif %}</pre>
```

Comment:
This kind of sucks, but this page doesn't have the ability to do something like "key, value, additional bits". It's probably worth reworking this to push more figuring logic into the view.